README.md

```diff
@@ -82,7 +86,56 @@ yaml_xml_list$body
 #> [10] <paragraph>\n  <text>Note that the </text>\n  <code>echo = FALSE</c ...
 ```
 
+### Inserting new elements
+
+You can insert new elements into the document via {xml2}, but you should make
+sure that the namespace matches that of your xml document. For example, let's
+say we wanted to add a new R code chunk after the setup chunk:
 
+> NOTE: Inserting new code MUST have a newline character at the end of the
```

Comment:
why, btw?

Reply:
I'm not sure! I think it has something to do with the stylesheet because it's accessible before it hits `xslt::xml_xslt()`. I think it may be worthwhile to open an issue to figure out what's going on, but probably not too high priority IMO.